cypress/e2e/trigger_builder.cy.js

```diff
@@ -81,20 +81,6 @@ const checkTriggerEditorUIValues = (trigger) => {
         .contains('Any interface')
         .should('be.selected');
     } else {
-      cy.get('#triggerInterfaceName')
-        .scrollIntoView()
-        .should('be.visible')
-        .contains(iface)
-        .should('be.selected');
-      cy.get('#triggerInterfaceMajor')
-        .scrollIntoView()
-        .should('be.visible')
-        .contains(simpleTrigger.interface_major)
-        .should('be.selected');
-      cy.get('#triggerPath')
-        .scrollIntoView()
-        .should('be.visible')
-        .and('have.value', simpleTrigger.match_path);
```

Comment:
those ones check for important features of the trigger editor, like the presence of an interface name and major.
Why remove them?